buildspec-pr.yaml

```diff
@@ -5,11 +5,9 @@ version: 0.2
 phases:
   install:
     commands:
-      # Start docker daemon inside the container
-      - nohup /usr/bin/dockerd --host=unix:///var/run/docker.sock --host=tcp://127.0.0.1:2375 --storage-driver=overlay2&
-      - timeout 15 sh -c "until docker info; do echo .; sleep 1; done"
```

Comment:
Aren't these still needed to start the docker daemon?

Reply:
@eladb We decided to move this into the ecr proxy image as well - so no. the `start.sh` scripts takes care of it.

Reply:
I'll add a comment and maybe later we can think about renaming/exposing another script to make it more clear.